dashboard/app/views/regional_partners/show.html.haml

```diff
@@ -57,4 +57,49 @@
   = link_to 'Edit', edit_regional_partner_path(@regional_partner)
 \|
 = link_to 'Back', regional_partners_path
-
+%br
+%h2 Program Managers
+%table.table.table-hover.table-condensed.table-auto-width{:id => 'program-managers-table'}
+  %thead
+    %th Action
+    %th ID
+    %th Name
+    %th Email
+    %th Recent Sign In
+    %th Sign In Count
+    %th Registered Account At
+  %tbody
+    - @regional_partner.program_managers.each do |program_manager|
+      %tr
+        %td= link_to 'Remove', action: :remove_program_manager, program_manager_id: program_manager.id
+        %td= program_manager.id
+        %td= program_manager.name
+        %td= program_manager.email
+        %td= program_manager.current_sign_in_at.try(:strftime, '%F')
+        %td= program_manager.sign_in_count
+        %td= program_manager.created_at.strftime('%F')
+%p Enter a User Name or Email to Assign a Program Manager
+= form_tag url_for(action: 'search_program_manager'), method: 'get', class: 'form-inline', enforce_utf8: false do
+  = text_field_tag :search_term, params[:search_term], class: 'form-control'
+  %button.btn{type: 'submit'}
+    %i.fa.fa-search
+-if @users
+  %table.table.table-hover.table-condensed.table-auto-width{:id => 'assign-program-managers-table'}
```

Comment:
I think it's worth displaying the applied search filter (if one is present) here. Otherwise, since the above search input is editable, it won't always match the results and might be confusing. Here's an [example](https://github.com/code-dot-org/code-dot-org/blob/baa92de5a32ff97945011ba25ae1e9e84d8d9a71/dashboard/app/views/pd/teacher_application/manage.html.haml#L23-L26).